### ISSUE TITLE
Update policy documents to reflect house style vis-a-vis singular 3rd-person pronouns

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6858]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)| @glyph reported|
|-|-|
|Trac ID|trac#6858|
|Type|enhancement|
|Created|2013-11-29 23:20:33Z|
|Branch|https://github.com/twisted/twisted/tree/doc-gendered-pronouns-6858|

There are [ample justifications](https://thoughtstreams.io/glyph/they-lives/) if anyone's curious.

The relevant policy document is in `doc/core/development/policy/doc-standard.xhtml`.

See [#6857](https://github.com/twisted/twisted/issues/6857).

Attachments:

* [t6858.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/850/850f5d65ecf96d01e6660e42afe72867be0bdbb9/5e7b49cc267d30cff56c369f423eaab3d1943fd0.diff) (876 bytes) - added by Alex on 2013-11-30 00:33:36Z - 
* [t6858.2.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/850/850f5d65ecf96d01e6660e42afe72867be0bdbb9/a89fc82080350d64fa6e2ed6417dd1c4ee7d2e73.diff) (715 bytes) - added by Alex on 2013-12-01 15:21:35Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__6858 6858
type__enhancement enhancement
reporter__glyph glyph
priority__normal normal
milestone__ 
branch__branches_doc_gendered_pronouns_6858 branches/doc-gendered-pronouns-6858
branch_author__tomprince tomprince
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1385767233000000 1385767233000000
changetime__1387540284000000 1387540284000000
version__None None
owner__Alex Alex

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_2"></a>|@tomprince commented|
|-|-|

(In [40876]) Branching to doc-gendered-pronouns-6858.

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_3"></a>|@tomprince commented|
|-|-|

(In [40877]) Apply t6858.diff from Alex.

Refs: [#6858](https://github.com/twisted/twisted/issues/6858)

**Comment by twisted-trac** [comment]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)<a name="note_4"></a>|@hynek commented|
|-|-|

Hey Alex,

thank you for looking into this. Some minor nits:

 - There is a spurious whitespace in “(they/their/ them)”. 
 - You seem to want to start a new paragraph, but in HTML you’ll have to add `<p>` tags (or remove the empty line if you don’t want to).
 - Open question: do we want to link to some resource why we think it’s important? It would be nice if people got some wider education from our docs.

Other than that, I have no complains. Unless some native speaker has any other issues with it, I’d be glad if it would be merged after addressing the first two points.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl commented|
|-|-|

I don't like the phrasing (repeated use of "pronoun"). I propose:

"When referring to a hypothetical person (such as "a user of a website written with twisted.web"), gender neutral pronouns (they/their/them) should be used."

**Comment by twisted-trac** [comment]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_6"></a>|Alex commented|
|-|-|

I've addressed the paragraphy markup, the stray whitespace, and the wording.

hynek - do you have a thought about what type of resource would be approrpirate? Were you thinking something like glyph's thoughstream/my blog post, or an academic work on the subject?

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_7"></a>|@tomprince commented|
|-|-|

(In [40927]) Apply t6858.2.diff from Alex.

Refs: [#6858](https://github.com/twisted/twisted/issues/6858)

**Comment by twisted-trac** [comment]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)<a name="note_8"></a>|@hynek commented|
|-|-|

Since all issues have been addressed and we got one more opinion by a native speaker this can be merged.

**Comment by twisted-trac** [comment]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_9"></a>|Alex commented|
|-|-|

Just to be clear, there's nothing blocking this right?

**Comment by twisted-trac** [comment]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)<a name="note_10"></a>|@hynek set status to `closed`|
|-|-|

(In [41005]) Merge doc-gendered-pronouns-6858: Mandate ungendered in documentation

Author: Alex
Reviewer: hynek
Fixes: [#6858](https://github.com/twisted/twisted/issues/6858)

See https://thoughtstreams.io/glyph/they-lives/ for the reasons.
